network/load-balancer/how-to/manage-frontends-and-backends.mdx

```diff
@@ -24,15 +24,15 @@ When you [create your Load Balancer](/network/load-balancer/how-to/create-load-b
 3. Click the **Frontends** tab.
 4. Click the green **+** button.
     <Lightbox src="scaleway-add-frontends.webp" />
-5. Enter a **name**, **port** and [**SSL certificate(s)**](/network/load-balancer/how-to/create-self-signed-tls-certificate/#how-to-import-a-self-signed-certificate-with-scaleway-load-balancers) for your new frontend. Note that each frontend must listen on a different port.
+5. Enter a **name**, **port** and (if necessary) [**SSL certificate(s)**](/network/load-balancer/how-to/add-certificate) for your new frontend. Note that each frontend must listen on a different port. Note that each frontend must listen on a different port. For help configuring your frontend, see our [dedicated reference documentation](/network/load-balancer/reference-content/configuring-load-balancer).
 6. Choose between **using an existing backend** or **creating a new backend**.
     - If using an existing backend, click **Use an existing Backend** and select the backend of your choice from the dropdown list.
-    - If creating a new backend, click **Create a new Backend** and enter a name, protocol, port proxy, health check type, server IP(s) and advanced settings as required.
+    - If creating a new backend, click **Create a new Backend** and enter a name, protocol, port proxy, health check type, server IP(s) and advanced settings as required. For help configuring your frontend, see our [dedicated reference documentation](/network/load-balancer/reference-content/configuring-load-balancer)
```

Comment:
As we are providing a link to the reference page maybe we cans just say "Enter required information" rather than:

> enter a name, protocol, port proxy, health check type, server IP(s) and advanced settings as required

Reply:
OK, done